.github/renovate.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["github>podium-lib/renovate-presets:top-level-module"],
```

Comment:
I've put you wrong here. Trygve has created a presets repo for Finn: https://github.com/finn-no/renovate-presets